lib/zclient.c

```diff
@@ -971,6 +997,12 @@ int zapi_route_decode(struct stream *s, struct zapi_route *api)
 				STREAM_GET(&api_nh->gate.ipv6, s, 16);
 				STREAM_GETL(s, api_nh->ifindex);
 				break;
+			default:
```

Comment:
again no I want the switch statement to catch the added type, without this we will miss new nexthops being added to the system.